demo/components/PContextAccordionItem.vue

```diff
@@ -1,153 +0,0 @@
-<template>
-  <div class="p-context-accordion-item">
```

Comment:
Why was this component deleted?

Reply:
It's no longer needed since the fix from https://github.com/PrefectHQ/prefect-design/pull/588 went live; we can use the `PContextAccordionItem` component from `prefect-design` instead of duplicating it now.

Reply:
ah nice, forgot it was duplicated.